bluetooth/src/commonMain/kotlin/ContextualLogger.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ Copyright 2025 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.bluetooth
+
+import com.splendo.kaluga.logging.LogLevel
+import com.splendo.kaluga.logging.Logger
+
+class ContextualLogger(private val logger: Logger, private val tag: String, private val context: Map<String, Any?> = LinkedHashMap()) : Logger {
```

Comment:
tags like `Bluetooth Device 00:A0:50:1B:64:91 Service 0000180a-0000-1000-8000-00805f9b34fb Characteristic-00002a26-0000-1000-8000-00805f9b34fb` are hard to query.`

updated one would be more like
`Bluetooth Device 00:A0:50:1B:64:91` `<message> <context values>`